test/bootlint_test.js

```diff
@@ -59,7 +59,7 @@ exports.bootlint = {
             [],
             'should not complain when the normal simple HTML5 doctype is used.');
         test.deepEqual(lintHtml(utf8Fixture('doctype/html5-legacy.html')),
-            [],
+            ['Document declares a non-HTML5 DOCTYPE'],
```

Comment:
Shouldn't need to change this. Either E001's implementation needs tweaking or presumably parse5 has a minor bug.